src/ice_shelf/MOM_ice_shelf_dynamics.F90

```diff
@@ -400,6 +402,7 @@ subroutine initialize_ice_shelf_dyn(param_file, Time, ISS, CS, G, US, diag, new_
   type(diag_ctrl), target, intent(in)    :: diag !< A structure that is used to regulate the diagnostic output.
   logical,                 intent(in)    :: new_sim !< If true this is a new simulation, otherwise
                                                  !! has been started from a restart file.
+  real,                    intent(in)    :: Cp_ice !< Heat capacity of ice (J kg-1 K-1)
```

Comment:
I suspect that `Cp_ice` is being passed around in rescaled units of [Q C-1 ~> J kg-1 degC-1], like `CS%Cp_ice`.

Reply:
This looks to have been resolved:
```
real :: Cp_ice !< The heat capacity of fresh ice [Q C-1 ~> J kg-1 degC-1].
```